blog/electron-16-0.md

```diff
@@ -0,0 +1,88 @@
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+Starting with Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron will be changing supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://www.electronjs.org/docs/latest/tutorial/electron-versioning) for more detailed information about versioning in Electron.
+
+### Stack Changes
+
+* Chromium `96`
+    * [New in Chrome 96](https://developer.chrome.com/blog/new-in-chrome-96/)
+* Node.js `16.9.1`
+    * [Node 16.9.1 blog post](https://nodejs.org/en/blog/release/v16.9.1/)
+* V8 `9.6`
+    * [V8 9.6 blog post](https://v8.dev/blog/v8-release-96)
+
+### Highlight Features
+
+* Now supports WebHID. [#30213](https://github.com/electron/electron/pull/30213)
+* Add data parameter to `app.requestSingleInstanceLock` to share data between instances. [#30891](https://github.com/electron/electron/pull/30891)
+* Pass securityOrigin to media permissions request handler. [#31357](https://github.com/electron/electron/pull/31357)
+* Add `commandLine.removeSwitch`. [#30933](https://github.com/electron/electron/pull/30933)
+
+See the [16.0.0 release notes](https://github.com/electron/electron/releases/tag/v16.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+Below are breaking changes introduced in Electron 16. More information about these and future changes can be found on the [Planned Breaking Changes](https://www.electronjs.org/docs/latest/breaking-changes) page.
+
+### Building Native Modules
+
+Building native modules for old versions of Electron (<=13, < 14.2.0, < 15.3.0) will require passing `--force-process-config` to node-gyp. You are likely to encounter this issue if your project is using custom scripts to build native modules. You can use [#2497](https://github.com/nodejs/node-gyp/pull/2497) to automatically pass `--force-process-config`.
+
+### Behavior Changed: `crashReporter` implementation switched to Crashpad on Linux
+
+The underlying implementation of the `crashReporter` API on Linux has changed from Breakpad to Crashpad, bringing it in line with Windows and Mac. As a result of this, child processes are now automatically monitored, and calling `process.crashReporter.start` in Node child processes is no longer needed (and is not advisable, as it will start a second instance of the Crashpad reporter).
+
+There are also some subtle changes to how annotations will be reported on Linux, including that long values will no longer be split between annotations appended with `__1`, `__2` and so on, and instead will be truncated at the (new, longer) annotation value limit.
+
+### API Changes
+
+There were no API changes in Electron 16.
```

Comment:
What exactly count as API changes? I would have considered some of the highlight features to be API changes:

> * Pass securityOrigin to media permissions request handler. [#31357](https://github.com/electron/electron/pull/31357)
> * Add `commandLine.removeSwitch`. [#30933](https://github.com/electron/electron/pull/30933)

Reply:
Hmm, but both of those are already merged into 15-x-y. Does it make sense to announce them as new here?

I do agree there's an ambiguity here, since it's not like we have blog posts for semver/minor releases, so API changes that get backported never get a blog entry. So maybe we should think about how to handle that properly, but IMO that's out-of-scope for this PR. We're documenting API changes here, and neither of these are changes relative to 15-x-y :man_shrugging:

Reply:
I would say "anything new since the last release blogpost" is relevant, but maybe then they shouldn't be API changes as much as "updates" or "noteworthy changes".